package.json

```diff
@@ -12,6 +12,7 @@
   "scripts": {
     "storybook": "start-storybook -p 6006 -c storybook",
     "storybook:build": "build-storybook -c storybook -o storybook/public",
+    "storybook:build-chromatic": "yarn storybook:build --webpack-stats-json",
```

Comment:
`--webpack-stats-json` is the secret sauce that chromatic requires to detect which files have code changes